fe/src/main/java/org/apache/doris/common/Pair.java

```diff
@@ -25,7 +27,9 @@
 public class Pair<F, S> {
     public static PairComparator<Pair<?, Comparable>> PAIR_VALUE_COMPARATOR = new PairComparator<>();
 
+    @SerializedName(value = "first")
```

Comment:
I'am not sure this is ok, cause there is no guarantee that the `F` and `S` object can also be serialized by GSON

Reply:
Users guarantee this when use Pair class？like Map and List.

Reply:
I add a comment.
When using Pair for persistence, users need to guarantee that F and S can be serialized through Gson